doc/admin/deployment.md

```diff
@@ -75,6 +75,69 @@ for latest information and examples.
 At this point of the process, the servers: and provider: section of the yaml
 file can be left blank and will be populated in the subsequent sections.
 
+#### Auto generate configuration file
+
+DAOS can attempt to produce a server configuration file that makes optimal use
+of hardware on a given set of hosts through the 'dmg config generate' command:
+
+```bash
+$ dmg config generate --help
+ERROR: dmg: Usage:
+  dmg [OPTIONS] config generate [generate-OPTIONS]
+
+Application Options:
+...
+  -l, --host-list=  comma separated list of addresses <ipv4addr/hostname:port>
+...
+
+[generate command options]
+      -p, --num-pmem=  Minimum number of SCM (pmem) devices required per
+                       storage host in DAOS system
```

Comment:
Is "storage host" the correct term here? I think this could be confusing for a reader who is still learning about DAOS. For consistency with the rest of the guide, I think we should use the term "I/O Server".

Reply:
We don't have multiple pmems per "I/O Server", I believe host is the right term here.

Reply:
Ah, yes. You're right, I was reading too quickly.